middle_end/flambda2/to_cmm/to_cmm_primitive.ml

```diff
@@ -100,6 +100,12 @@ let block_load ~dbg (kind : P.Block_access_kind.t) (mutability : Mutability.t)
   | Values { field_kind = Immediate; _ } ->
     C.get_field_computed Immediate mutability ~block ~index dbg
   | Naked_floats _ -> C.unboxed_float_array_ref block index dbg
+  | Mixed { field_kind; _ } -> (
+    (* CR mixed blocks: a backend person should confirm I can just use these and
```

Comment:
This and the below CR should be fine, at least until we allow packed int32#s, etc.

Note `unboxed_float_array_ref` always generates a mutable load, which is overly conservative. (It doesn't matter much)